cli/testScript.sh

```diff
@@ -47,7 +47,7 @@ HARDHAT_CONFIG=./build/hardhat.config.js node build/ts/index.js genProofs \
     -w true 
 HARDHAT_CONFIG=./build/hardhat.config.js node build/ts/index.js proveOnChain \
     --poll-id 0 \
-    --proof-dir proofs/
+    --proof-dir proofs/ \
```

Comment:
?

Reply:
that needed to be there for the next line to be considered part of the command, otherwise it would have been counted as a command of its own and result in a bash shell error

Reply:
basically something we missed on a previous commit

Reply:
my fault sorr TT